### ISSUE TITLE
Arabic

### ISSUE THREAD

**Issue by Rabios** [dlang-tour/core#757]
No arabic documentation available for D, But if i have time i can help offering one...

**Comment by PetarKirov** [comment]
Hey @Rabios thank you for your interest! I don't have linguistics background, but I understand that there are different [varieties of Arabic](https://en.m.wikipedia.org/wiki/Varieties_of_Arabic) and I'm sure how to proceed. Should there be a translation for each (major) variation? In which variety/dialect are you interested in providing a translation?
For example, we have Portuguese and Brazilian Portuguese as separate transactions, so I suppose we can follow the same pattern.

Be sure to check our guide to contributing: https://github.com/dlang-tour/core/blob/master/CONTRIBUTING.md

**Comment by Rabios** [comment]
@PetarKirov Yes that's right, Use al-Fuṣḥā dialect, It's dialect can be understood by anyone who speak any arabic dialect, So you won't get troubled when you use a dialect like khaliji (Arab gulf countries), Or any other arab dialects!

**Comment by PetarKirov** [comment]
Thanks @Rabios for info! I suppose that's what is described on [wikipedia](https://en.wikipedia.org/wiki/Modern_Standard_Arabic) as Modern Standard Arabic.

I have created the repo here: https://github.com/dlang-tour/arabic and I have added you to the @dlang-tour/arabic so you're now able to start the translation!

**Comment by Rabios** [comment]
OK, Let's do it!

**Comment by Rabios** [comment]
@PetarKirov  Oh...Mind if you take a look at arabic dlang tour repo? I've done translations for some pages!

**Comment by Rabios** [comment]
Forget that, I can merge them myself :)
